Source/CommandLine.cpp

```diff
@@ -409,8 +409,25 @@ static ArgumentList createCommandLineArgs (String commandLine)
                                                 || argList.containsOption ("--run-tests");
 
         if (! hasValidateOrOtherCommand)
-            if (isPluginArgument (argList.arguments.getLast().text))
-                argList.arguments.insert (argList.arguments.size() - 1, { "--validate" });
+        {
```

Comment:
I don't think you need these changes [412 - 430].
All this code should do is if "--validate" is not in the args AND the last argument looks like a plugin path (`isPluginArgument`), then insert a `--validate` argument before it.

The actual "fix" for the original issue is below on line 482.